tsconfig.json

```diff
@@ -11,7 +11,7 @@
 		"allowJs": true,
 		"baseUrl": ".",
 		"paths": {
-			"@clayui/*": ["node_modules/@clayui/*/src"],
+			"@clayui/*": ["packages/clay-*/src"],
```

Comment:
Have you ever tested if this works well? by running `yarn build` on packages and checking if the `lib` folder is being generated correctly with types.

Reply:
oh my bad, I didn't mean to include this. This was the previous code before you updated it because I was running into issues.